mne_connectivity/utils/utils.py

```diff
@@ -99,6 +141,60 @@ def seed_target_indices(seeds, targets):
     return indices
 
 
+def seed_target_multivariate_indices(seeds, targets):
+    """Generate indices parameter for multivariate seed-based connectivity.
+
+    Parameters
+    ----------
+    seeds : array-like of array-like of int
+        Seed indices.
+
+    targets : array-like of array-like of int
+        Target indices.
+
+    Returns
+    -------
+    indices : tuple of array of array of int
+        The indices padded with the invalid channel index ``-1``.
```

Comment:
I agree that -1 is technically a valid index and np.nan would be best, however np.nan is a float and cannot be represented in an array of ints.

If we switched to an array of floats, np.nan could be represented, however numpy does not like floats as indices, so we would have to convert to ints every time we want to index channels.

One alternative is numpy masked arrays (https://numpy.org/doc/stable/reference/maskedarray.html) however I'm not sure how this would be handled when saving using h5netcdf. Perhaps another option could be to choose a very unlikely integer like -99,999.

Ultimately I would be happy to go with whatever you feel is most appropriate.

Reply:
I'll defer to you. I think it's something we can consider changing in the future if it's an issue for users.